config/global.go

```diff
@@ -63,6 +66,14 @@ func (tlc *TopLevelConfig) GetGlobal() (*Global, error) {
 		return nil, err
 	}
 
+	if global.TTL == 0 {
```

Comment:
Ah. I wondered about that, was being defensive. Ok, I will remove them 
and re-run the tests.

On 8 Mar 2016, at 15:48, Madhav Puri wrote:

>> @@ -63,6 +66,14 @@ func (tlc *TopLevelConfig) GetGlobal() (*Global, 
>> error) {
>> 		return nil, err
>> 	}
>>
>> +	if global.TTL == 0 {
>
> I think we can safely skip setting defaults here again as 
> `NewGlobalConfig()` already sets the defaults and `json.Unmarshal` 
> will not touch these values if they are not set.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/171/files#r55449898